src/Instances.js

```diff
@@ -32,19 +35,19 @@ const languages = [
 // as those are pulled from the backend
 const filterConfig = [
   {
-    label: 'Resource Type',
+    label: <FormattedMessage id="ui-inventory.instances.resourceType" />,
```

Comment:
Please, check if there is no prop type warnings regarding to new type of the labels.

Reply:
They are, so this PR is to resolve this:
https://github.com/folio-org/stripes-components/pull/789